config/owasp/suppressions.xml

```diff
@@ -78,4 +78,67 @@
     <notes>Temporary suppression for commons-fileupload-1.4.jar</notes>
     <cve>CVE-2023-24998</cve>
   </suppress>
+  <suppress>
```

Comment:
Do we still need these after Paul's update yesterday?

Reply:
Fixed that